src/hotspot/share/utilities/copy.hpp

```diff
@@ -300,7 +300,6 @@ class Copy : AllStatic {
  protected:
   inline static void shared_disjoint_words_atomic(const HeapWord* from,
                                                   HeapWord* to, size_t count) {
-
```

Comment:
I don't think this justify to change the file.

Reply:
Reverted.

Reply:
Un-done.  Thanks.